lib/smack/SmackRep.cpp

```diff
@@ -32,6 +39,9 @@ const Expr* SmackRep::NUL = Expr::id(NULL_VAL);
 const string SmackRep::STATIC_INIT = "$static_init";
 
 const int SmackRep::width = 0;
+const unsigned SmackRep::ptrsize = 32;
```

Comment:
The variable is used in SmackRep::getPtrSize() which returns the size of element the pointer points to. When the type of element is aggregate, I have some difficulty determining the size. I tried to google it while nothing useful is found. Currently hardcode the return value of getPtrSize() when the argument is of aggregate type works well.